velero/mysql/20-statefulset.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: mysql
+  namespace: mysql
+spec:
+  selector:
+    matchLabels:
+      app: mysql
+      env: prod
+  serviceName: mysql
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        backup.velero.io/backup-volumes: data
+        pre.hook.backup.velero.io/command: '["/sbin/fsfreeze", "--freeze", "/var/lib/mysql"]'
+        pre.hook.backup.velero.io/container: fsfreeze
+        post.hook.backup.velero.io/command: '["/sbin/fsfreeze", "--unfreeze", "/var/lib/mysql"]'
+        post.hook.backup.velero.io/container: fsfreeze
+        # pre.hook.backup.velero.io/command: "['/usr/bin/mysql', '-e', '\"flush tables with read lock;\"']"
+        # pre.hook.backup.velero.io/container: mysql
+        # post.hook.backup.velero.io/command: "['/usr/bin/mysql', '-e', '\"unlock tables;\"']"
+        # post.hook.backup.velero.io/container: mysql
+      labels:
+        app: mysql
+        env: prod
+    spec:
+      serviceAccountName: mysql
+      securityContext:
+        runAsUser: 0
+        fsGroup: 999
+      containers:
+      - name: mysql
+        image: mysql:5.7
+        env:
+        - name: MYSQL_ALLOW_EMPTY_PASSWORD
+          value: "1"
+        ports:
+        - name: mysql
+          containerPort: 3306
+        volumeMounts:
+        - name: data
+          mountPath: /var/lib/mysql
+          subPath: mysql
+        - name: conf
+          mountPath: /etc/mysql/mysql.conf.d
+        resources:
+          requests:
+            cpu: 500m
+            memory: 1Gi
+      - name: fsfreeze
+        image: ubuntu:bionic
```

Comment:
Is there any chance that a smaller ubuntu image also has the fsfreeze command. 

It is a small detail that we don't really need to change, but if a slim image can do the same, it is always better. A bionic image can be pretty large.